### ISSUE TITLE
Require python for runtime because of fetch-ocsp-response

### ISSUE THREAD

**Issue by drscream** [nghttp2/nghttp2#992]
The readme describe that Python is only required for the python binding. It's also still possible to build the `nghttp2` code without the binding. But from what I see is that at runtime python is required because of the `fetch-ocsp-response` script. Is that correct?

**Comment by tatsuhiro-t** [comment]
Yes, fetch-ocsp-response is written in python, and it requires Python in runtime.
